include/enums.h

```diff
@@ -258,6 +258,13 @@ enum eMouseEventType {
 	MOUSE_SCROLLED_DOWN
 };
 
+enum eKeyboardEventType {
```

Comment:
I would suggest enum classes for everything. This increases type safety and eliminates name clases. The values no longer have to be prepended with the enum name. So in this case, that would be:
`enum class eKeyboardEventType {
    None,
    Hold,
    Pressed
};`
Usage is then: `eKeyboardEventType::None`.

Reply:
Oh that is awesome. I did not know there was such a thing! It was really bugging me to have these enums clash with each other. `enum classes` will solve that for sure. Thanks!